### ISSUE TITLE
Document not (yet) implemented features

### ISSUE THREAD

**Issue by frederikschubert** [mgechev/angular2-hot-loader#5]
I read your article on your blog (http://blog.mgechev.com/2015/10/26/angular2-hot-loader-hot-loading-tooling/) and saw that there are some missing features that you documented.
I think it would be helpful to create issues for them here in this repo to gain more insight in the current capabilities and shortcomings of the hot-loader.
I don't know whether the list you gave in your article is still valid and wanted to ask you which of those are already implemented.

**Comment by mgechev** [comment]
Yes, I think it is out-of-date. More specifically, which features do you mean?

**Comment by frederikschubert** [comment]
I think the patching of the component's constructor does not work yet.
Also I read something in your code regarding the `@RouteConfig` decorator. Does this fix the issue you mentioned in your article?
I don't know if it is possible but it would be nice to load new components/files that have been created after the page has been loaded.
Also thank you for your great work! This project would make developing angular applications much more enjoyable.

**Comment by mgechev** [comment]
Thanks for the kind words! Yes, constructors doesn't work yet. Haven't tested it with `@RouteConfig` but it may actually work with the latest changes. I think the entire metadata associated to the component is being updated.

> I don't know if it is possible but it would be nice to load new components/files that have been created after the page has been loaded.

Should be possible. At this point my main focus is to achieve stability of the current set of features and to make the code more readable :-)

Would you open issues for the features you'd like to use? Also if you'd enjoy it - feel free to open another PR with new features/bug fixes.

**Comment by frederikschubert** [comment]
Alright! I think I will spend some time playing around with the hot-loader during the holidays :). Then I will really see which features are needed and will create issues (and if I can PRs) for them.
